api/restHandler/AppWorkflowRestHandler.go

```diff
@@ -309,6 +358,20 @@ func (handler *AppWorkflowRestHandlerImpl) GetWorkflowsViewData(w http.ResponseW
 		common.WriteJsonResp(w, err, nil, http.StatusBadRequest)
 		return
 	}
+	v := r.URL.Query()
+	envIdsString := v.Get("envIds")
+	envIds := make([]int, 0)
+	if len(envIdsString) > 0 {
+		envIdsSlices := strings.Split(envIdsString, ",")
+		for _, envId := range envIdsSlices {
+			id, err := strconv.Atoi(envId)
+			if err != nil {
+				common.WriteJsonResp(w, err, "please provide valid envIds", http.StatusBadRequest)
+				return
+			}
+			envIds = append(envIds, id)
+		}
+	}
```

Comment:
you can make a utility function to do this

Reply:
done